tonic/io.py

```diff
@@ -52,11 +55,11 @@ def read_dvs_128(filename):
     all_events = get_aer_events_from_file(filename, data_version, data_start)
     all_addr = all_events["address"]
     t = all_events["timeStamp"]
-
+  
     x = (all_addr >> 8) & 0x007F
     y = (all_addr >> 1) & 0x007F
     p = all_addr & 0x1
-
+  
```

Comment:
not sure about all this whitespace, could you please use black? https://pypi.org/project/black/

Reply:
I did not know about black. Will do!